### PR TITLE
Release candidate for v10.5 + Removing AIO network template

### PR DESCRIPTION
**Pull request by nimrody15**

### CODE REVIEW

File: aws/DRVault-Single-Deployment.json [hunk 1]
```diff
@@ -1230,60 +1230,60 @@
...
-                "PSM": "ami-7cf8d797",
-                "PSMP": "ami-acc8e747"
+                "Vault": "ami-070990fb0a4adcb1c",
+                "Components": "ami-0b7366d5a32014f5f",
+                "CPM": "ami-04fece7e673892d18",
+                "PVWA": "ami-0d78264adddbf0e09",
+                "PSM": "ami-0d7a9a6ebd8620ba7",
+                "PSMP": "ami-0552713509028f263"
             },
             "eu-west-2": {
```
Comment (zivshits) [inline aws/DRVault-Single-Deployment.json]: EU-WEST-2 appears twice.  see line 1240.

File: aws/Full-PAS-Deployment.json [hunk 2]
```diff
@@ -3137,60 +259,60 @@
...
-                "PSM": "ami-7cf8d797",
-                "PSMP": "ami-acc8e747"
+                "Vault": "ami-070990fb0a4adcb1c",
+                "Components": "ami-0b7366d5a32014f5f",
+                "CPM": "ami-04fece7e673892d18",
+                "PVWA": "ami-0d78264adddbf0e09",
+                "PSM": "ami-0d7a9a6ebd8620ba7",
+                "PSMP": "ami-0552713509028f263"
             },
             "eu-west-2": {
```
Comment (zivshits) [inline aws/Full-PAS-Deployment.json]: double eu-west-2 entry. first time at line 269

File: aws/PAS-AIO-dr-Deployment.json [hunk 3]
```diff
@@ -2222,60 +2222,60 @@
...
-                "PSM": "ami-7cf8d797",
-                "PSMP": "ami-acc8e747"
+                "Vault": "ami-070990fb0a4adcb1c",
+                "Components": "ami-0b7366d5a32014f5f",
+                "CPM": "ami-04fece7e673892d18",
+                "PVWA": "ami-0d78264adddbf0e09",
+                "PSM": "ami-0d7a9a6ebd8620ba7",
+                "PSMP": "ami-0552713509028f263"
             },
             "eu-west-2": {
```
Comment (zivshits) [inline aws/PAS-AIO-dr-Deployment.json]: double eu-west-2 entry.  first time at line 2232

File: aws/PAS-AIO-network-environment-template.json [hunk 4]
```diff
@@ -1,841 +0,0 @@
-{
```
Comment (zivshits) [inline aws/PAS-AIO-network-environment-template.json]: Why do we remove AIO network?

File: aws/PAS-AIO-template.json [hunk 5]
```diff
@@ -1687,60 +1687,60 @@
...
-                "PSM": "ami-7cf8d797",
-                "PSMP": "ami-acc8e747"
+                "Vault": "ami-070990fb0a4adcb1c",
+                "Components": "ami-0b7366d5a32014f5f",
+                "CPM": "ami-04fece7e673892d18",
+                "PVWA": "ami-0d78264adddbf0e09",
+                "PSM": "ami-0d7a9a6ebd8620ba7",
+                "PSMP": "ami-0552713509028f263"
             },
             "eu-west-2": {
```
Comment (zivshits) [inline aws/PAS-AIO-template.json]: double eu-west-2 entry. First time at line 1697

File: aws/PAS-Component-Single-Deployment.json [hunk 6]
```diff
@@ -2304,60 +2304,60 @@
...
-                "PSM": "ami-7cf8d797",
-                "PSMP": "ami-acc8e747"
+                "Vault": "ami-070990fb0a4adcb1c",
+                "Components": "ami-0b7366d5a32014f5f",
+                "CPM": "ami-04fece7e673892d18",
+                "PVWA": "ami-0d78264adddbf0e09",
+                "PSM": "ami-0d7a9a6ebd8620ba7",
+                "PSMP": "ami-0552713509028f263"
             },
             "eu-west-2": {
```
Comment (zivshits) [inline aws/PAS-Component-Single-Deployment.json]: double entry eu-west-2. See line 2314

File: aws/Vault-Single-Deployment.json [hunk 7]
```diff
@@ -1217,60 +1217,60 @@
...
-                "PSM": "ami-7cf8d797",
-                "PSMP": "ami-acc8e747"
+                "Vault": "ami-070990fb0a4adcb1c",
+                "Components": "ami-0b7366d5a32014f5f",
+                "CPM": "ami-04fece7e673892d18",
+                "PVWA": "ami-0d78264adddbf0e09",
+                "PSM": "ami-0d7a9a6ebd8620ba7",
+                "PSMP": "ami-0552713509028f263"
             },
             "eu-west-2": {
```
Comment (zivshits) [inline aws/Vault-Single-Deployment.json]: double entry eu-west-2. See line 1227

File: aws/DRVault-Single-Deployment.json (hunk 1 above)
Comment (nimrody15) [reply aws/DRVault-Single-Deployment.json]: Good catch.
Fixed

File: aws/PAS-AIO-dr-Deployment.json (hunk 3 above)
Comment (nimrody15) [reply aws/PAS-AIO-dr-Deployment.json]: fixed

File: aws/PAS-AIO-template.json (hunk 5 above)
Comment (nimrody15) [reply aws/PAS-AIO-template.json]: fixed

File: aws/PAS-Component-Single-Deployment.json (hunk 6 above)
Comment (nimrody15) [reply aws/PAS-Component-Single-Deployment.json]: fixed

File: aws/Vault-Single-Deployment.json (hunk 7 above)
Comment (nimrody15) [reply aws/Vault-Single-Deployment.json]: fixed

File: aws/PAS-AIO-network-environment-template.json (hunk 4 above)
Comment (nimrody15) [reply aws/PAS-AIO-network-environment-template.json]: That was a decision made in v10.3 not release this network, it is not secured according to CyberArk standard.

**Pull request merged**